packages/builders/scripts/fast-usdc/start-fast-usdc.build.js

```diff
@@ -15,7 +15,7 @@ import { parseArgs } from 'node:util';
 /**
  * @import {CoreEvalBuilder, DeployScriptFunction} from '@agoric/deploy-script-support/src/externalTypes.js'
  * @import {ParseArgsConfig} from 'node:util'
- * @import {FastUSDCConfig, FeedPolicy} from '@agoric/fast-usdc/src/types.js'
+ * @import {FastUSDCConfig, FeedPolicy} from '@agoric/fast-usdc';
```

Comment:
Nit: there's a semicolon here but not for the lines above. I wonder if eslint/prettier should have a rule for this

Reply:
I don't think ESlint has a linter for it. There's a Prettier plugin for JSDoc but it's quite buggy